filer/tests/admin.py

```diff
@@ -245,6 +248,8 @@ def test_file_upload_no_duplicate_files(self, extra_headers={}):
         )
         self.assertEqual(Image.objects.count(), 1)
         self.assertIn('error', response.content)
+        self.assertIn('already exists in the clipboard', response.content)
```

Comment:
Nitpick. It would be really nice if the error message would be stored in a field on the admin class. 
That way this test won't break when the message changes.